Car Simulator/Assets/Scripts/CameraRecorder.cs

```diff
@@ -75,21 +104,37 @@ private Texture2D RTImage(Camera camera)
         RenderTexture.active = currentRT;
         return image;
     }
-
+ 
     // Update is called once per frame
     private void Update()
     {
         _timeSinceLastCapture += Time.deltaTime;
-        if(_timeSinceLastCapture > 1 / _framesPerSecond)
+        _currentFrameTimestamp = Time.time;
+        if (_timeSinceLastCapture > 1 / _framesPerSecond)
         {
+            string newPhotosUpdateInfo = _carController.mapId.ToString() + " " + _carController.carId.ToString() + " "
```

Comment:
```suggestion
            string newPhotosUpdateInfo = "screen" + _carController.mapId.ToString() + " " + _carController.carId.ToString() + " "
```
Unity was supposed to send photo update with prefix "screen". Is this correct place?